app/js/components/tabs/tabcomponents/drugOrderComponent.jsx

```diff
@@ -118,11 +119,19 @@ class DrugOrderComponent extends Component {
                                 <span className="inline-label">From:</span>
                             </div>
                             <div className="col-sm-3">
-                                <input className="form-control" type="date" name="from-date" />
+                                <DatePicker
+                                    dateFormat="DD-MM-YYYY"
+                                    className="form-control"
+                                    name="from-date"
+                                />
                             </div>
                             <span className="inline-label">To:</span>
                             <div className="col-sm-3">
-                                <input className="form-control" name="to-date" type="date" />
+                                <DatePicker
+                                    dateFormat="DD-MM-YYYY"
```

Comment:
This describes the format that the date picker view would display as shown here
![image](https://cloud.githubusercontent.com/assets/23452705/26306437/46ad9c16-3eeb-11e7-9283-4c07a7ebed9d.png)
so the user has an idea of the format to enter the date value. I don't think it's necessary to allow the user specify what format he/she wants to enter the date in when the format accepted is clearly stated. What do you think about this @dkayiwa